internal/schema/db_loader_test.go

```diff
@@ -0,0 +1,126 @@
+// Copyright 2021 Zenauth Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+package schema_test
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"io"
+	"io/fs"
+	"io/ioutil"
+	"net/url"
+	"os"
+	"path/filepath"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/cerbos/cerbos/internal/schema"
+	"github.com/cerbos/cerbos/internal/storage"
+	"github.com/cerbos/cerbos/internal/storage/db/sqlite3"
+	"github.com/cerbos/cerbos/internal/test"
+	"github.com/cerbos/cerbos/internal/util"
+)
+
+func TestDBLoader(t *testing.T) {
```

Comment:
We can roll these tests into https://github.com/cerbos/cerbos/blob/4859fc2bad31b64862caa16f450fb89047e94833/internal/schema/schema_test.go#L30 by adding an extra loop to iterate through the different store implementations

Reply:
I have come up with something, let me know if it looks good to you.